docs/install/harvester-configuration.md

```diff
@@ -574,8 +582,8 @@ install:
 
 Harvester v1.2.0 ships with four addons:
 
-- harvester-vm-import-controller
-- harvester-pcidevices-controller
+- vm-import-controller (chartName: `harvester-vm-import-controller`)
+- pcidevices-controller (chartName: `harvester-pcidevices-controller`)
 - rancher-monitoring
 - rancher-logging
```

Comment:
We have a new one rancher-vcluster (experimental)

Reply:
there is also a seeder addon:

https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rancherd-22-addons.yaml
```
  - apiVersion: harvesterhci.io/v1beta1
    kind: Addon
    metadata:
      name: harvester-seeder
      namespace: harvester-system
    labels:
      addon.harvesterhci.io/experimental: "true"
    spec:
  ```

Reply:
The `chartName` comes for the first time in this file, we may add an explanation of `chartname` here, then say it is same as addon name by default, for those who have different names, they are listed separately.

Reply:
For `vcluster-rancher`, @ibrokethecloud mentioned, it is not packed into Harvester OS, namely, not possible to be enabled via the `harvester-configuration`.

Need to include it in the coming `vcluster-rancher` chapter in the Harvester Doc.

Reply:
great mention :smile: :+1: - I've updated this in the latest force push  :smile: